padle/padle.py

```diff
@@ -733,26 +736,26 @@ async def advance(self, ctx):
     @padleadmin.command()
     async def filter(self, ctx):
         """Re-creates the list of monsters"""
-        confirmation = await get_user_confirmation(ctx, "This command will take a long time to execute. Run anyways?")
-        if not confirmation:
-            return
         dbcog = await self.get_dbcog()
         mgraph = dbcog.database.graph
         final = []
-        # await ctx.send(mgraph.max_monster_id)
-        for i in range(1, 8800):  # hard coded bc max_monster_id as calculated above is 15k???
-            monster = await dbcog.find_monster(str(i), ctx.author)
-            with suppress(Exception):
-                nextTrans = mgraph.get_next_transform(monster)
-                prevTrans = mgraph.get_prev_transform(monster)
-                if (monster is not None and monster.name_en is not None and monster.on_na and
-                        monster.series.series_type != 'collab' and
-                        ((monster.sell_mp >= 50000 and (monster.superawakening_count > 1 or prevTrans is not None)) or
-                         "Super Reincarnated" in monster.name_en) and not monster.is_equip and
-                        nextTrans is None and monster.level >= 99):
-                    final.append(str(i))
+        for i in range(1, mgraph.max_monster_id):
```

Comment:
Is this an exclusive or inclusive cap? You're using it as an exclusive cap.

E.g. Does `max_monster_id=1234` mean that monster_ids stop before 1234, or that 1234 is the last known one?

Reply:
FYI, it is an inclusive cap.

Reply:
So it needs to be +1.

Is there a better way to iterate through the monster DB?